projects/observability/src/shared/components/gauge/gauge.component.ts

```diff
@@ -8,14 +8,15 @@ import { Arc, arc, DefaultArcObject } from 'd3-shape';
     <div class="gauge-container" (htLayoutChange)="this.onLayoutChange()">
       <svg class="gauge" *ngIf="this.rendererData">
         <g attr.transform="translate({{ rendererData.origin.x }}, {{ rendererData.origin.y }})">
-          <path class="gauge-ring" [attr.d]="rendererData.backgroundArc" />
+          <path class="gauge-ring" [attr.d]="rendererData.backgroundArc" *ngIf="rendererData.radius > 80" />
```

Comment:
should this use the same constant as line 19 below?

Reply:
Yeah. I missed it. Will update